src/shogun/machine/Machine.h

```diff
@@ -333,6 +333,22 @@ class CMachine : public CStoppableSGObject
 			return false;
 		}
 
+		/** Continue Training
+		 *
+		 * This method can be used to continue a prematurely stopped
+		 * call to CMachine::train.
+		 * It throws an error if the machine does not support this feature.
+		 *
+		 * NOT IMPLEMENTED!
```

Comment:
As mentioned in another earlier comment from Heiko iirc, is "NOT IMPLEMENTED!" "inherited" in the sublasses documentation (where this method may be actually implemented)?

Reply:
it is only if sublcasses dont write a new docstring.
But as said, I would remove this here